packages/k8s/tests/prepare-job-test.ts

```diff
@@ -130,6 +131,17 @@ describe('Prepare job', () => {
     expect(got.spec?.containers[2].args).toEqual(['-c', 'sleep 60'])
   })
 
+  it('should not throw exception using kube scheduler', async () => {
+    // only for rwx volumes or single node cluster
```

Comment:
```suggestion
    // only for ReadWriteMany volumes or single node cluster
```
I think you wanted to say this right? :relaxed: